packages/sdk/src/utils/Mapping.ts

```diff
@@ -45,9 +53,9 @@ export class Mapping<K extends LookupKeyType, V> {
      **/
     constructor(opts: CacheMapOptions<K, V> | LazyMapOptions<K, V>) {
         if ('maxSize' in opts) {
-            this.delegate = new LRU<string, Item<K, V>>({
-                maxSize: opts.maxSize,
-                maxAge: opts.maxAge
+            this.delegate = new LRUCache<string, Item<K, V>>({
+                max: opts.maxSize,
```

Comment:
Would it make sense to use `maxSize` option of LRUCache instead of the `max` option? Both of these options seem to limit the item count, but `max` option allocates the storage pre-emptively. This may be more performant, but also bit wasteful, as very typically our caches are not in anywhere near the maximum limit.

Reply:
I guess. That's how it used to be. I'll adjust.